hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1129,7 +1129,18 @@
   public static final int MASTER__META_TRANSITION_HANDLER_COUNT_DEFAULT = 1;
 
   /** Conf key for enabling meta replication */
+  /**
+   * @deprecated Since 2.4.0, will be removed in 4.0.0. Please enable meta replica HedgedRead mode
+   * by setting "hbase.locator.meta.replicas.mode" to "HedgedRead".
+   */
+  @Deprecated
   public static final String USE_META_REPLICAS = "hbase.meta.replicas.use";
```

Comment:
Though I do not like this config as I said before its scope is too wide, but since it is there for a very long time, I do not think the new config could replace it? The config only controls how we locate regions, but this controls all the requests to meta table...

Reply:
Makes sense. Let me not to retire it as there are more families now supported besides "info". The config part will check this config knob if ReplicaMode is not configured.